upf/upf_pfcp_api.c

```diff
@@ -1620,7 +1623,7 @@ handle_create_far (upf_session_t * sx, pfcp_create_far_t * create_far,
 	if (ISSET_BIT (far->forwarding_parameters.grp.fields,
 		       FORWARDING_PARAMETERS_BBF_APPLY_ACTION))
 	  {
-	    if (far->forwarding_parameters.bbf_apply_action &=
+	    if (far->forwarding_parameters.bbf_apply_action &
```

Comment:
this change is in the "checkstyle fixes" commit, but is actually a functional change

Reply:
commits updated